homeassistant/components/anova/config_flow.py

```diff
@@ -0,0 +1,32 @@
+"""Config flow for Anova."""
+from __future__ import annotations
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN
+
+
+class AnovaConfligFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Sets up a config flow for Anova."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, str] | None = None
+    ) -> FlowResult:
+        """Handle a flow initiated by the user."""
+        if user_input is not None:
+            await self.async_set_unique_id(user_input["device_unique_id"])
+            self._abort_if_unique_id_configured()
```

Comment:
We need to validate that we can connect and it's a valid device id as well here before creating the entry

Reply:
Added!